spark-3.1-spanner-lib/pom.xml

```diff
@@ -29,20 +29,6 @@
       <artifactId>google-cloud-spanner</artifactId>
     </dependency>
 
-    <dependency>
-      <groupId>io.netty</groupId>
-      <artifactId>netty-tcnative-boringssl-static</artifactId>
-      <version>2.0.20.Final</version> <!-- See table for correct version -->
-      <scope>runtime</scope>
-    </dependency>
-
-    <dependency>
```

Comment:
Does the test pass if we remove the junit dependency ?

Reply:
JUnit is added by the parent